packages/quick_actions/quick_actions_ios/ios/Classes/ShortcutItemServicing.swift

```diff
@@ -0,0 +1,15 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import UIKit
+
+/// Services `UIApplicationShortcutItem`s.
+protocol ShortcutItemServicing: AnyObject {
```

Comment:
From https://docs.google.com/document/d/1H8F-0OmJ5GF4BCwfC_j7rs7k02dLkrcxpe2IZbaQg5Q/edit?disco=AAAAjPbIzyk

> ShortcutItemServicing isn't a great name, it's not "servicing" anything (which doesn't make much sense).  In this case it's a property bag with a single `shortcutItems` property.  That sounds like protocol describing what something is, like `ShortcutItemCollection`, rather than a capability protocol.

Reply:
`ShortcutItemCollection` sounds like just a dumb collection without any behaviors. But setting the items actually change the system behavior. 

I think `AppShortcutControlling` can be a good name to capture this behavior, because it's something that **controlls** the app shortcut behavior.